### ISSUE TITLE
Missing headers on OSX

### ISSUE THREAD

**Issue by gordonwoodhull** [sabel83/metashell#17]
Sorry if there is an obvious explanation for this. It may also have to do with the homebrew "bottle". 

I installed metashell via homebrew on Mavericks, which was easy, but I can't execute the first line in the metaparse tutorial:

```
#include <mpllibs/metaparse/string.hpp>
```

First it complained about missing `unistd.h`. I stubbed that out because I know it's obsolete. But then it complained about `errno.h` and I don't know if that's obsolete.

I don't really understand if metashell is using its own headers or how to configure that. So I'm back to using files and the compiler to try out metaparse. :-)

**Comment by r0mai** [comment]
You can help metashell detect include paths, by passing `-I<directory>` parameters.

Still, I would advise you to try building the latest version (i.e master from github). A number of things has been improved since 2.0.0:
http://metashell.readthedocs.org/en/latest/getting_metashell/build/#building-metashell-on-os-x

Let us know, if you still have questions/issues :)

**Comment by sabel83** [comment]
You might also try the installer you can download from this site instead of homebrew.

Apart from fixing the include path on OS X, you should still be able to try the Metaparse tutorial in the online demo of Metashell (http://abel.web.elte.hu/shell), where it is configured (you can even include the headers to skip the first sections if needed).

**Comment by gordonwoodhull** [comment]
Thanks @r0mai, @sabel83. I didn't want to point to another compiler's header files, and I tried online metashell, which is very cool but slow and pasting is indirect (too bad about this browser security restriction).

In my regular work I am creating a [tool for data scientists to do exploratory data analysis](https://github.com/att/rcloud). I think the same principle applies here: anything that is slow or adds extra steps, gets in the way of doing exploration. So I really want a locally running metashell where there are no extra steps to paste code, and it runs as fast as my computer will allow it.

Looks like metashell is just about finished building while I type this. I'll report back once I've tried it out!
